taxi/tests/test_models.py

```diff
@@ -0,0 +1,47 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Car
+
+
+class ModelTests(TestCase):
+    def test_manufacturer_str(self):
+        manufacturer = Manufacturer.objects.create(
+            name="Test Manufacturer", country="Test Country"
+        )
+        self.assertEqual(
+            str(manufacturer), f"{manufacturer.name} {manufacturer.country}"
+        )
+
+    def test_driver_str(self):
+        driver = get_user_model().objects.create(
+            first_name="Test First Name",
+            last_name="Test Last Name",
+            username="Test Username",
+            license_number="Test License NUmber",
```

Comment:
There's a typo in the `license_number` value: 'Test License NUmber'. Ensure that this value is consistent with the expected format and validation logic.